src/Runner.Worker/ExpressionManager.cs

```diff
@@ -1,162 +0,0 @@
-﻿using System;
```

Comment:
Trying to move all of this kind of stuff into PipelineTemplateEvaluator. Ideally condition in job message would be TemplateToken not string (serializer compatible)